### PR TITLE
Http client ignore/match; micrometer 1.6.2

### PR DESCRIPTION
**Pull request by ebullient**

### CODE REVIEW

**Review by kenfinnigan — APPROVED** [review]

**Comment by gsmet** [comment]
@ebullient @kenfinnigan the test failure looks related.

**Comment by ebullient** [comment]
y. tags are emitted in a different order in that one build.

**Comment by gsmet** [comment]
@ebullient just a reminder that substantial changes for 1.11 needs to be in by tonight (can be tonight your time as long as CI is green when I wake up tomorrow morning).

File: extensions/micrometer/runtime/src/main/resources/META-INF/quarkus-extension.yaml [hunk 1]
```diff
@@ -4,9 +4,8 @@ metadata:
   - "micrometer"
   - "metrics"
   - "metric"
-  - "prometheus"
   - "monitoring"
   guide: "https://quarkus.io/guides/micrometer-metrics"
   categories:
   - "observability"
-  status: "experimental"
+  status: "stable"
```
Comment (kenfinnigan) [inline extensions/micrometer/runtime/src/main/resources/META-INF/quarkus-extension.yaml:11]: woohoo!

**Review by kenfinnigan — APPROVED** [review]

**Pull request merged**